randovania/data/json_data/prime3.txt

```diff
@@ -3812,12 +4678,12 @@ Asset id: 1808913861159744919
 Command Courtyard
 Asset id: 18221971457754325372
 > Dock to Courtyard Passage; Heals? False
-  * Normal Door to Courtyard Passage/Dock to Command Courtyard
+  * Plasma Door to Courtyard Passage/Dock to Command Courtyard
   > Bottom Floor
       Trivial
 
 > Dock to Lift Hub; Heals? False
-  * Normal Door to Lift Hub/Dock to Command Courtyard
+  * Plasma Door to Lift Hub/Dock to Command Courtyard
   > Top Floor
       Trivial
 
```

Comment:
- Connect to Courtyard Passage with requirement "After Barrier Event"
- Connect to Energy Cell as Trivial
- Connect to Flux Control with requirement "After Fan Event"